oolab/src/main/java/agh/ics/oop/GrassField.java

```diff
@@ -0,0 +1,93 @@
+package agh.ics.oop;
+
+import java.util.*;
+
+import static java.lang.Integer.*;
+
+public class GrassField extends AbstractWorldMap{
+    private List<Grass> grasses = new ArrayList<>();
+
+    private List<Grass> nDistinctRandomGrasses(int n){
+        Random r = new Random(78543266);
+        Set<Vector2d> vectors = new HashSet<>();
+        while(vectors.size() != n){
+            vectors.add(new Vector2d(r.nextInt(0, (int) Math.sqrt(10*n)),  r.nextInt(0, (int) Math.sqrt(10*n))));
```

Comment:
a co z powtórzeniami?

Reply:
Wydaje mi się, że powtórzenia są eliminowane z samego faktu, że jest to set i takie było założenie podczas pisania tego algorytmu.